src/components/CardWithContent/index.tsx

```diff
@@ -17,38 +19,75 @@ export default function CardWithContent({
   title,
   children: content,
 }: Props) {
+  const { colorMode } = useColorMode();
+  const isDarkTheme = colorMode === 'dark';
+  
+  // Create a theme that adapts to dark/light mode
+  const theme = createTheme({
```

Comment:
Is this the same code as in the previous file? If yes, maybe this can be put in its own file and imported? So all the properties `MuiCard`, `MuiAccordion`, etc... are under the same object